budcamp/Homework3/Homework3/task2/TestResults.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework3
+{
+    internal class TestResults
+    {
+        private Dictionary<string, int> results;
```

Comment:
As you can see in the problem requirements, here is an example of test result:

Id: 2312 
Topic: math
Date: 12/01/2023 
Result: 12/20 (60%)

You're currently only storing the topicName and score in this dictionary. According to requirements you should store more information about your result. You should probably create another class, like the following:
```
internal class TestResult {
   public List<Test> Tests { get; set; }
   public string Topic { get; set; }
   public int CorrectAnswers { get; set; }
   public DateTime DateOfTest { get; set; }
}
```

Then this field will change to:
```
private Dictionary<string, TestResult> results;
```